docs/general/server/media/shows.md

```diff
@@ -120,17 +122,17 @@ Supported folder types are:
 
 ```txt
 Shows
-└── Series (2010)
+└── Series Name (2010)
     ├── Season 01
-    │   ├── Episode S01E01.mkv
-    │   ├── Episode S01E02.mkv
+    │   ├── Series Name S01E01.mkv
+    │   ├── Series Name S01E02.mkv
     │   ├── featurettes
     │   │   └── Some Featurette.mkv
     │   └── interviews
     │       └── Interview with the Director.mp4
     ├── Season 02
-    │   ├── Episode S02E01.mkv
-    │   ├── Episode S02E02.mkv
+    │   ├── Series Name S02E01.mkv
+    │   ├── Series Name S02E02.mkv
```

Comment:
I thought using episodes names was the preferred way no?

Reply:
No. The parser is written in a way that for example:
```
- Stargate (1990)
-- Season 01
--- Stargate S01E01.mkv
```
Gets parsed first and most reliably